cli/tests/test_next_prev_commands.rs

```diff
@@ -634,8 +632,7 @@ fn test_prev_editing() {
     ◆  zzzzzzzzzzzz
     "###);
 
-    // --edit is implied when already editing a non-head commit
-    let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["prev"]);
+    let (stdout, stderr) = test_env.jj_cmd_ok(&repo_path, &["prev", "--edit"]);
```

Comment:
nit: perhaps, this part can be removed at all. `prev --edit` is tested above.

Reply:
Done.